README.md

```diff
@@ -0,0 +1,18 @@
+# Noboty Practice Bot Code
+
+> Code for the 2018 practice bot
+
+## Noboty?
+Noboty is a play on words from Team 118's Everybot. We called it this because noboty will use it.
+
+## Controls
+We used a one-joystick control system for sake of convenience. Moving the joystick on the Y axis moves the robot forwards or backwards, and the X axis causes it to rotate.
+
+#### Sarah mode
+This mode cuts the speed of the robot in half, for precision movement in confined areas as well as a simple safety precaution for inexperienced drivers. Sarah mode was created when Sarah, a member of our team, smashed the robot into a wall at full speed. It is activated by pressing the red button on the joystick.
+
+## Authors
+[See contributors](https://github.com/frc1418/noboty/graphs/contributors)
+
+## Licensing
+In-season, use of this software is restricted by the FRC rules. After the season ends, the MIT License applies instead.
```

Comment:
This doesn't apply to this repository as it's not a competition robot. Say what we say here: https://github.com/frc1418/tbapy